### PR TITLE
Abehern/vz 7299 add a vz release menu to the open source documenation landing page

### PR DESCRIPTION
**Pull request by AbeHern**
Added custom dropdown menu to Docs landing page. May still need to dynamically update the docs version that is currently selected.

### CODE REVIEW

File: layouts/partials/navbar.html [hunk 1]
```diff
@@ -3,6 +3,22 @@
         <a class="navbar-brand" href="{{ .Site.Home.RelPermalink }}">
 		<span class="navbar-logo">{{ if .Site.Params.ui.navbar_logo }}{{ with resources.Get "icons/logo.svg" }}{{ ( . | minify).Content | safeHTML }}{{ end }}{{ end }}</span>
 	</a>
+	<div class="release-documentation-container">
+		<div class="release-dropdown-container">
+			<button class="release-dropdown-button">
+				v1.4
```
Comment (Abhijithang) [inline layouts/partials/navbar.html:9]: Can you make this dynamically populated using the variables in params.toml config file? It's preferably required to update the new version releases only in config file i.e., params.toml.

File: config/_default/params.toml [hunk 2]
```diff
@@ -112,3 +112,7 @@ enable = false
 	icon = "fab fa-slack"
         desc = "Chat with other project developers"
 
+# Release dropdown menu urls and version numbers
+[releaseDropDown]
+urlReleaseList = ["https://verrazzano.io/latest/docs/", "https://verrazzano.io/v1.3/docs/", "https://verrazzano.io/v1.2/docs/", "https://verrazzano.io/v1.1/docs/", "https://verrazzano.io/archive/docs/"]
```
Comment (Abhijithang) [inline config/_default/params.toml:117]: Can you use variable to populate the public url?

**Comment by anders-swanson** [comment]
Can you add a screenshot of how this looks?

File: assets/scss/_styles_project.scss [hunk 3]
```diff
@@ -1,3 +1,96 @@
...
+    display: inline-flex;
+    width: 0;
+    height: 0;
+    border-left: 7px solid transparent;
+    border-right: 7px solid transparent;
+    border-top: 7px solid white;
+}
+
+.release-dropdown-divider{
+    width:5px;
```
Comment (anders-swanson) [inline assets/scss/_styles_project.scss:63]: no spaces here

Comment (AbeHern) [reply assets/scss/_styles_project.scss:63]: Screen Shot 2022-12-16 at 3 04 24 PM

File: config/_default/params.toml (hunk 2 above)
Comment (pshirodkar-orcl) [inline config/_default/params.toml:117]: https://verrazzano.io/archive/docs/ displays list of versions which seem to be dynamically generated. Hardcoding will make this maintenance nightmare. Can you please take a look?

Comment (AbeHern) [reply config/_default/params.toml:117]: Addressing it, will be updating it

File: config/_default/params.toml [hunk 4]
```diff
@@ -112,3 +112,7 @@ enable = false
 	icon = "fab fa-slack"
         desc = "Chat with other project developers"
 
+# Release dropdown menu urls and version numbers
+[releaseDropDown]
+releaseVersionsList = ["v1.4", "v1.3", "v1.2", "v1.1", "archive"]
```
Comment (pshirodkar-orcl) [inline config/_default/params.toml:117]: We should not have this hardcoded list either.

File: layouts/partials/navbar.html [hunk 5]
```diff
@@ -3,6 +3,31 @@
...
+			<div class="release-dropdown-content">
+				{{ $vzDocsPath := .Site.Params.releaseDropDown.vzDocumentPath }}
+				{{ $versionNumbers := .Site.Params.releaseDropDown.releaseVersionsList }}
+				{{ range $versionIndex, $releaseVersionNumber := $versionNumbers }}
+					{{ $versionUrl := (printf "%s%s%s"  $vzDocsPath $releaseVersionNumber "/docs") }}
+					{{ if eq $versionIndex 0 }}
+						{{ $latestVersionUrl := (printf "%s%s%s"  $vzDocsPath "latest" "/docs") }}
+						<a href="{{ $latestVersionUrl }}">{{ printf "%s%s" $releaseVersionNumber " (latest)" }}</a>
+					{{ else if eq $releaseVersionNumber "archive" }}
+						<a href="{{ $versionUrl }}">All Versions</a>
```
Comment (pshirodkar-orcl) [inline layouts/partials/navbar.html:23]: What is the purpose of "All Versions"?

**Review by anders-swanson — APPROVED** [review]

**Review by Abhijithang — APPROVED** [review]

File: config/_default/config.toml [hunk 6]
```diff
@@ -30,3 +30,23 @@ resampleFilter = "CatmullRom"
 quality = 75
 anchor = "smart"
 
+# Release versions including link to archive
+[[params.versions]]
+version = "v1.4 (latest)"
+url = "https://verrazzano.io/latest/docs"
```
Comment (pshirodkar-orcl) [inline config/_default/config.toml:36]: This should be https://verrazzano.io/v1.4/docs

**Pull request merged**